rdt-open-reader/src/main/java/io/ona/rdt/viewholder/PatientRegisterViewHolder.java

```diff
@@ -62,6 +63,7 @@ public void getView(Cursor cursor, SmartRegisterClient client, RegisterViewHolde
 
         final Patient patient = new Patient(patientName, sex, baseEntityId, patientId,
                 Integer.parseInt(patientAge));
+        patient.setDob(Utils.getValue(commonPersonObjectClient.getColumnmaps(), Constants.DBConstants.DOB, false));
```

Comment:
Why not have this as part of the constructor?

Reply:
done